content/en/docs/setup/install/multicluster/single-network/primary/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Add a Primary Cluster
+description: Describes how to add a primary cluster to your mesh on a flat network.
+weight: 20
+keywords: [multicluster, primary-cluster, deployment]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+To build a {{< gloss >}}multicluster{{< /gloss >}} deployment, you can add
+{{< gloss "primary cluster" >}}primary clusters{{< /gloss >}} to your
+{{< gloss >}}service mesh{{< /gloss >}}.
+
+To give you clean instructions, this section starts right after you complete the
+[initial configuration](/docs/setup/install/multicluster/single-network/initial-configuration),
+and assumes that `Cluster_1` and `Cluster_2` reside on the same network.
+However, you can add primary clusters to your existing mesh regardless of your
+current [deployment model](/docs/ops/deployment/deployment-models/).
```

Comment:
I think we can leave deployment model out of this paragraph. It's not really adding much and is just adding to the link/definition overload of the reader.

Reply:
Linking allows us to avoid information overload by not having to define all those concepts on this page. Readers don't need to click the link.